packages/shared/src/schema/judge/index.ts

```diff
@@ -1 +1,3 @@
+export * from './common';
```

Comment:
This shouldn't be exported; it should only be used by the flies in this directory not those outside of it

Reply:
but removing this export gives this error on the common.test.ts file:
< **Property 'commonSchema' does not exist on type 'typeof import("/Users/aa253074/Dev/Hangar/packages/shared/src/schema/judge/index")'.** >

Reply:
You'll just need to update your import to pull from the actual file instead of from the barrel file; see my comment below